File: eval/fixtures/legacy-directory-backed-spring-commerce/src/main/java/com/acme/legacy/billing/BillingRefundController.java
package com.acme.legacy.billing;

@RestController
@RequestMapping("/api/v5/billing")
public class BillingRefundController {
  private final BillingRefundService billingRefundService = new BillingRefundService();

  @PostMapping("/refunds/{refundId}/audit")
  public ResponseEntity<?> auditRefund(String refundId) {
    return ResponseEntity.ok(billingRefundService.rebuildRefundAudit(refundId));
  }
}

File: eval/fixtures/legacy-directory-backed-aspnet-commerce/src/Legacy/Billing/BillingRefundController.cs
namespace Acme.Legacy.Billing;

[ApiController]
[Route("api/v6/billing")]
public class BillingRefundController : ControllerBase
{
  private readonly BillingRefundService _billingRefundService = new BillingRefundService();

  [HttpPost("refunds/{refundId}/audit")]
  public IActionResult AuditRefund(string refundId)
  {
    return Ok(_billingRefundService.RebuildRefundAudit(refundId));
  }
}
